test/docfx.Test/DocfxTest.cs

```diff
@@ -64,20 +61,20 @@ public static async Task Run(TestData test, DocfxTestSpec spec)
             finally
             {
                 t_repos.Value = null;
-                t_cachePath.Value = null;
-                t_statePath.Value = null;
+                t_appDataPath.Value = null;
             }
         }
 
-        private static (string docsetPath, string cachePath, string statePath, string outputPath, Dictionary<string, string> repos)
+        private static (string docsetPath, string appDataPath, string outputPath, Dictionary<string, string> repos)
             CreateDocset(TestData test, DocfxTestSpec spec)
         {
             var testName = $"{Path.GetFileName(test.FilePath)}-{test.Ordinal:D2}-{HashUtility.GetMd5HashShort(test.Content)}";
-            var basePath = Path.GetFullPath(Path.Combine(spec.Temp ? Path.GetTempPath() : "docfx-test", testName));
+            var basePath = Path.GetFullPath(Path.Combine(spec.Temp ? Path.GetTempPath() : "docfx-tests", testName));
```

Comment:
This change is needed to due to cache folder structure change. It allows tests to succeed on server.